frontend/src/pages/ErrorsV2/ErrorBody/components/ErrorOccurenceDate/index.tsx

```diff
@@ -13,17 +13,18 @@ const ErrorOccurenceDate = ({ errorGroup }: Props) => {
 			gap="4"
 			alignItems="center"
 			flexShrink={0}
-			flexWrap="nowrap"
+			flexWrap="wrap"
+			maxWidth="full"
+			style={{ rowGap: 8 }}
```

Comment:
@ccschmitz Is this the right way to approach this?

Reply:
We could also add this as a property in our sprinkles and allow it to accept our default space values. Would be similar to how we support `gap`:

https://github.com/highlight/highlight/blob/28138c5bc40addf98aec4162d1c9fd211141ffe7/packages/ui/src/css/sprinkles.css.ts/#L88

Reply:
💯 I need to do a deep dive into Sprinkles.